### DISCUSSION TITLE
Raspbian install through snap incomplete and fails to start

### DISCUSSION THREAD

**Discussion by karl-baker** [PurpleI2P/i2pd#1850] [General]
Hello,

I saw that my raspbian did an upgrade to version 2.45.1 through snap. Since then the service is not running and is unable to start.

In the meantime I removed the previous and current versions, and tried a clean installation. Not all expected files are installed.

Has anyone had the same experience?

Thanks.

**Comment by r4sas** [comment]
Looks like snap's `base` option broken it. I've seen same reports, investigating now.

**Reply by karl-baker** [reply]
Thanks

**Reply by r4sas** [reply]
Please try edge package
`snap install --edge i2pd`

**Reply by karl-baker** [reply]
That fixed the issue, thanks for your help.
